### ISSUE TITLE
LibreSSL does not return version when an incompatible config file can be found

### ISSUE THREAD

**Issue by TinCanTech** [OpenVPN/easy-rsa#603]
Required:
```
verify_ssl_lib() {
	if [ -z "$EASYRSA_SSL_OK" ]; then
		save_OPENSSL_CONF="$OPENSSL_CONF"
		unset OPENSSL_CONF
		# redirect std-err to ignore missing etc/ssl/openssl.cnf file
		"$EASYRSA_OPENSSL" version
		val="$("$EASYRSA_OPENSSL" version 2>/dev/null)" || die "\
SSL version error: $val
"
		case "${val%% *}" in
			# OpenSSL does require a safe config-file for ampersand
			OpenSSL) ssl_lib=openssl; require_safe_ssl_conf=1 ;;
			LibreSSL) ssl_lib=libressl; require_safe_ssl_conf=1 ;;
			*) die "\
Missing or invalid OpenSSL: ${val%% *}
Expected to find openssl command at: $EASYRSA_OPENSSL"
		esac

		OPENSSL_CONF="$save_OPENSSL_CONF"
		unset save_OPENSSL_CONF

```

**Comment by TinCanTech** [comment]
https://github.com/libressl-portable/portable/issues/767

**Comment by TinCanTech** [comment]
This is not a bug in `easyrsa` because `$OPENSSL_CNF` always points to `$EASYRSA_SAFE_CONF`.  However, do **not** copy `openssl-easyrsa.cnf` to `/usr/local/etc/ssl`, unless the intention is to _bork_ LibreSSL.

**Comment by TinCanTech** [comment]
Closed via Commit a3db60f229514abda9c301780dd0bdf4fb2976d1
